### ISSUE TITLE
Can we have a 1.0 release?

### ISSUE THREAD

**Issue by demiankatz** [ghislainf/zf2-whoops#21]
This code hasn't changed in quite a while but a release hasn't been tagged since 2013; it would be nice if we had a released version we could pin in composer.json instead of relying on dev-master. Is there any reason we can't do this? If there are known issues holding up a release, I'd be willing to contribute some time to help resolve them.

**Comment by func0der** [comment]
Should be quick to do, shouldn't it?

**Comment by demiankatz** [comment]
I'm not sure if it's worth continuing to work with this project, given that Zend Framework is now fully deprecated in favor of Laminas. It looks like somebody has forked and updated the project here, so that might be the best place to go for future Whoops integration: https://github.com/Ppito/laminas-whoops

**Comment by func0der** [comment]
But a simple 1.0 release that is published to packigist would not hurt. Even if it is only to "abandon: true". Then we would have the security that nothing will change.

**Comment by demiankatz** [comment]
@func0der, good point, that would be worth doing; I'm just not sure if any of the owners of this repository are still watching. Perhaps this conversation will attract a little attention. :-)
